oceanbench/broadcast_permanent.go

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
Does this file need to be here still or should this be / is this part of OceanTV?

Reply:
Permanent broadcasts are not (yet) managed by OceanTV.
@saxon-milton

Reply:
They are :)